<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseCatalogue</class>
 <widget class="QWidget" name="BrowseCatalogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>731</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #384b06;;</string>
   </property>
   <widget class="QWidget" name="ItemsBG" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>651</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #face23;
border-radius: 20px;</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>631</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="itemsLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="searchBarBG" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9ebe41;
border-radius: 15px</string>
    </property>
    <widget class="QLineEdit" name="searchBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>631</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Serif</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #face23;
border-radius: 10px;
font: bold 12pt;
padding-left: 10px;
color: #364e1b;
</string>
     </property>
     <property name="text">
      <string>Search here...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 75, 6)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
